js/components/PictureList/PictureList.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { View, Text, Image, ListView, Dimensions } from 'react-native';
+import { styles } from './styles';
+
+var { height, width } = Dimensions.get('window');
+
+class PictureList extends Component {
+  constructor(){
+    super();
+    const ds = new ListView.DataSource({rowHasChanged: (r1, r2) => r1 !== r2});
+    this.state = {
+      dataSource: ds.cloneWithRows(this.props.list),
+    }
+  }
+  render() {
+    return (
+      <View>
+        <Image source={{uri: 'https://s-media-cache-ak0.pinimg.com/736x/73/91/86/7391863170be07a072ffeb3e7605db2a.jpg'}}
```

Comment:
Okay, but instead of a URL, we will most likely be using require with the path to an asset when they are received